### ISSUE TITLE
Hong Kong Airport

### ISSUE THREAD

**Issue by mdav43** [shooshx/BoardingBarcode#7]
@shooshx you legend. 

Arrived late to airport, airline counter closed and couldn’t get my already checked in boarding pass. 

Security wouldn’t let me it until I found your generator and did it myself. It worked at Hong Kong’s automated gates. 

You literally saved me $100s and a lot - a lot -  of inconvenience

**Comment by pinkiedukie** [comment]
Did you board the plane Without paying???

**Comment by tokigavandraren** [comment]
> Did you board the plane Without paying???

Hello. Yes. I did it several times, in Europe primarily. Nothing I would recommend tho for obvious reasons. But yes. It works 9/10 times.

**Comment by tokigavandraren** [comment]
Yes. Correct: Dates, airports, flight nr., and class is what matters.
